docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
+services:
+  test_with_rvm:
+    build:
+      context: .
+      dockerfile: ./with_rvm.dockerfile
+    environment:
+      GITHUB_TOKEN: $GITHUB_TOKEN
```

Comment:
What is the relationship between this and `HOMEBREW_GITHUB_API_TOKEN`? I added the following .env file to the repo but it still required me to do pass the token into the individual commands
```
# .env
HOMEBREW_GITHUB_API_TOKEN=xxxxx
GITHUB_OAUTH_CREDENTIALS=xxxxx:x-oauth-basic
GITHUB_TOKEN=xxxxx
```

Reply:
This was misconfigured. I've updated the file to use HOMEBREW_GITHUB_API_TOKEN instead, and verified on my local that it works.